packages/react-providers/src/client/use-client-reducer.ts

```diff
@@ -275,12 +300,17 @@ export const useClientReducer = ({ env, client, signer, options, census3Options
   // dispatch helper methods
   const clear = () => dispatch({ type: ClientClear })
   const createAccount = () => dispatch({ type: ClientAccountCreate })
-  const errorAccount = (error: ErrorPayload) => dispatch({ type: ClientAccountError, payload: error })
+  const errorAccount = (error: ErrorPayload) => dispatch({ type: ClientAccountFetchError, payload: error })
```

Comment:
Shouldn't this be errorAccountFetch?

Reply:
Yes, it should.. and I was about to change it but it also implies changing a lot more definitions that weren't changed when they should (my fault). I also did not realize I did commit some of these changes. I think I'll undo them.

Reply:
Ah, actually, this one is a correct change, because we had both `ClientAccountError` and `ClientAccountFetchError` defined. I just removed the `ClientAccountError` one and used the expected `ClientAccountFetchError` instead. This doesn't mean this does not need to be refactored... all "basic" errors refer to the `fetch` method, and they should be renamed likewise.